src/modules/groupMethods.ts

```diff
@@ -219,6 +219,28 @@ export const groupMethods = {
         return memberships.map((m) => ({ ...m, group: addCalculatedGroupFields(m.group, sessionUser) }));
     },
 
+    getTreeMembershipsCount: async (id: string) => {
+        const treeViewGroups: { id: string }[] = await prisma.$queryRaw`
+        WITH RECURSIVE group_hierarchy AS (
+            SELECT * FROM "Group" WHERE "parentId" = ${id}
+            UNION ALL
+            SELECT g.* FROM "Group" g
+            INNER JOIN group_hierarchy gh ON gh.id = g."parentId"
+        )
+        SELECT id FROM group_hierarchy;
+    `;
+
+        const subtreeTotalGroups = await prisma.membership.count({
+            where: {
+                groupId: {
```

Comment:
What to do with archived memberships? They probably should not be counted. This could be tested. Base functionality could also be tested

What to do with deactivated users? Probably should be included, since they are visible in the list.